controller/volume_rebuilding_controller.go

```diff
@@ -379,11 +379,26 @@ func (vbc *VolumeRebuildingController) reconcile(volName string) (err error) {
 		}
 	}()
 
+	existingVol := vol.DeepCopy()
+	defer func() {
+		if err != nil {
+			return
+		}
+		if reflect.DeepEqual(existingVol.Status.Conditions, vol.Status.Conditions) {
+			return
+		}
+
+		if _, err = vbc.ds.UpdateVolume(vol); err != nil {
```

Comment:
Should be `UpdateVolumeStatus`?
```suggestion
		if _, err = vbc.ds.UpdateVolumeStatus(vol); err != nil {
```

Reply:
Yes, you are right.
Modified to use `UpdateVolumeStatus`.
Thanks.